django/db/models/fields/files.py

```diff
@@ -75,6 +75,7 @@ def open(self, mode='rb'):
             self.file.open(mode)
         else:
             self.file = self.storage.open(self.name, mode)
+        return self
```

Comment:
There doesn't seem to be a single test for FieldFile, probably because it's mostly an internal class used by FileField. I've only added the "return self" here as well because I've added it to all overrides in subclasses of File.